winter/http/throttling.py

```diff
@@ -46,9 +46,6 @@ def allow_request(self, request, view) -> bool:
         history = self.cache.get(key, [])
         now = time.time()
 
-        while history and history[-1] <= now - throttling_.duration:
-            history.pop()
```

Comment:
@berenda-andrey Please review it carefully.

Reply:
What for have you removed it?
It is for counting user requests. It is from rest framework

Reply:
I failed to tests it, that's why I decided to remove it as dead code. I will try again, maybe you can suggest a test case?

Reply:
I'll try too and after that I'll ping you

Reply:
Fixed the test case and return the code.